### PR TITLE
Tasks page

### PR DESCRIPTION
**Pull request by TathagataChakraborti**
+ [x] Task tab on the landing page
+ [x] Task descriptions in [wiki](https://github.com/IBM/covid19-india-data/wiki/Challenge-Tasks).

### CODE REVIEW

**Comment by TathagataChakraborti** [comment]
@MayankAgarwal this is ready to merge once you go through the task descriptions on the task tab and the wiki. I will start sending the emails once it merges.

**Review by MayankAgarwal — APPROVED** [review]
looks good!

**Pull request merged**
